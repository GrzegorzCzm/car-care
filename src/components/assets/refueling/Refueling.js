import React, { Component } from 'react';


class Refueling extends Component {
    render() {
        return (
            <div className="Root">
                Refueling
      </div>
        );
    }
}

export default Refueling;
