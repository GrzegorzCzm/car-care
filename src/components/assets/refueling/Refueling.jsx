import React from 'react';


const Refueling = () => (
  <div className="Root">
    Refueling
  </div>
);

export default Refueling;
