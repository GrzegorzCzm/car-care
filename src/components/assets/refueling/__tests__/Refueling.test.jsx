import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Refueling from '../Refueling';


describe('Refueling', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Refueling />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
