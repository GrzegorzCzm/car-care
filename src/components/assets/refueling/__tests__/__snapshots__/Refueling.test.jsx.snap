// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Refueling renders correctly 1`] = `
<div
  className="Root"
>
  Refueling
</div>
`;
